### DISCUSSION TITLE
Add custom page types

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#9937] [Q&A / Support]
In backend сreate document "right click -> Add page -> Blank"
Can I create document with a predefined template?
So that I can add custom page types?

**Comment by dvesh3** [comment]
@esisl you can use predefined document types https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Predefined_Document_Types.html

**Reply by esisl** [reply]
I'm dumb.
Thank you! :)
